src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToStoreAProjectWithDependencies.cs

```diff
@@ -161,22 +161,23 @@ public void store_nativeonlyassets()
             storeDirectory.Should().OnlyHaveFiles(files_on_disk);
         }
 
-        [CoreMSBuildOnlyFact]
+        [CoreMSBuildOnlyFact(Skip = "https://github.com/dotnet/sdk/issues/2914")]
```

Comment:
Is this test netcoreapp3.0 specifc?

Reply:
@livarcocc Are you ok with disabling the store tests while we make a call. I'm a bit worried about taking on debt if we get the call to keep it.

Reply:
This one started failing for .NET Core 1.x.  I discussed disabling these tests with Livar, although I'm not sure I called out the fact that one of the failing ones wasn't for .NET Core 3.

Reply:
It's a little concerning that we broke 1.x with this change. I can't think of a reason why that would happen. I'd prefer that we investigate in case we're breaking other aspects of exisitng projects.

Reply:
I wasn't aware that we were breaking 1.x projects as well. I thought the breaking was for netcoreapp3.0 projects.